### PR TITLE
Minor fixes; Tenet Chapel Rename

### PR DESCRIPTION
**Pull request by ttgmichael**
Changes revelation to be lower case.
Deletes an extra 4 lines in QUESTLV100
Fixes typos.

I can also add Tenet Chapel -> Tenet Church change here. Let me know what people think! 

Kudos to @DoubleN94 for the suggestion https://github.com/Treeofsavior/EnglishTranslation/issues/1391

### CODE REVIEW

File: ITEM.tsv [hunk 1]
```diff
@@ -1960,7 +1960,7 @@ ITEM_20150317_001959	Crystal Collection
 ITEM_20150317_001960	Mausoleum Collection
```
Comment (Sunaries) [inline ITEM.tsv]: Royal Mausoleum

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -1710,9 +1706,9 @@ QUEST_LV_0100_20150317_001705	You passed the last test.{nl}Congratulations.{nl}P
 QUEST_LV_0100_20150317_001706	Please go to the forest again.{nl}The person waiting for you is there.{nl}Please put the pieces of the scupture toghther at the designated place.
 QUEST_LV_0100_20150317_001707	Please do not forget your valuable experiences here.{nl}Don't be afraid, please move forward.
 QUEST_LV_0100_20150317_001708	You woke me up to take the test.
-QUEST_LV_0100_20150317_001709	I've been waiting for you. I'm the Swordsman Master, the one who will help you. {nl} I'm not sure if you know about attribute training. {nl}
-QUEST_LV_0100_20150317_001710	Attributes can give additional abilities to stats and skills. {nl} You can learn attributes from each class master by paying them silver.
-QUEST_LV_0100_20150317_001711	You'll be able to fight easier if you learn attributes. {nl}
+QUEST_LV_0100_20150317_001709	I've been waiting for you. I'm the Swordsman Master. I can help you develop your swordsmanship {nl} Do you know about attribute training? {nl}
+QUEST_LV_0100_20150317_001710	Attributes can grant additional abilities to Abilities and Skills. {nl} You learn attributes from each class master by paying the training fee in Silver.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:1710]: Abilities=Statuses
also, capitalizing silver?

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -2351,17 +2347,17 @@ QUEST_LV_0100_20150317_002346	Strange Sack
...
 QUEST_LV_0100_20150317_002357	But, disasters are happening just as I saw them to be. {nl}And it means that the hope, which I saw further away, will also definitely come. {nl}
-QUEST_LV_0100_20150317_002358	Dear Savior, our sister Saule is losing her light. {nl} Find her at the back of the light from the colorful valley and you will be guided to the next Revelation.
+QUEST_LV_0100_20150317_002358	Dear Savior, our sister Saule is losing her light. {nl} Find her at the back of the light from the colorful valley and you will be guided to the next revelation.
 QUEST_LV_0100_20150317_002359	$Revelation of Crystal Mine
-QUEST_LV_0100_20150317_002360	Dear Savior, {nl} in order for me to regain my whole appearance and stop the demons, {nl} you must find all the Revelations hidden in the axis of all times. {nl}
+QUEST_LV_0100_20150317_002360	Dear Savior, {nl} in order for me to regain my whole appearance and stop the demons, {nl} you must find all the revelations hidden in the axis of all times. {nl}
 QUEST_LV_0100_20150317_002361	{memo X}$But only you and I can save the world. {nl} I will wait for you in the high gardens.
 QUEST_LV_0100_20150317_002362	$Revelation of Tenet Chapel
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Missed this one.

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -7370,10 +7366,10 @@ QUEST_LV_0100_20150714_007365	Take some time to think about it
 QUEST_LV_0100_20150714_007366	The spirit with depression(1)
 QUEST_LV_0100_20150714_007367	{memo X}Notice/The bead of comfort has been perfected!#3
 QUEST_LV_0100_20150714_007368	The spirit with depression(2)
-QUEST_LV_0100_20150714_007369	The request of Siaulai Goddess(1)
+QUEST_LV_0100_20150714_007369	The Request of Goddess Saule(1)
 QUEST_LV_0100_20150714_007370	Ask him how you can help
 QUEST_LV_0100_20150714_007371	Looking at the thorny vines
-QUEST_LV_0100_20150714_007372	The request of Siaulai Goddess(2)
+QUEST_LV_0100_20150714_007372	The Request of Goddess Saule(2)
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:7372]: space between Saule and the bracket

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -7370,10 +7366,10 @@ QUEST_LV_0100_20150714_007365	Take some time to think about it
 QUEST_LV_0100_20150714_007366	The spirit with depression(1)
 QUEST_LV_0100_20150714_007367	{memo X}Notice/The bead of comfort has been perfected!#3
 QUEST_LV_0100_20150714_007368	The spirit with depression(2)
-QUEST_LV_0100_20150714_007369	The request of Siaulai Goddess(1)
+QUEST_LV_0100_20150714_007369	The Request of Goddess Saule(1)
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:7369]: space between Saule and the bracket

**Comment by Sunaries** [comment]
You have my vote for Tenet Church since it is more appropriate to use regarding the aesthetics of the area.

**Comment by ttgmichael** [comment]
Changed to Tenet Church

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**